src/main/java/io/github/enkarin/bookcrossing/books/dto/BookModelDto.java

```diff
@@ -48,7 +48,8 @@ public class BookModelDto {
 
     public Attachment getAttachment() {
         if (attachment != null) {
-            return new Attachment(attachment.getName(), attachment.getData(), attachment.getExpansion());
+            return new Attachment(attachment.getAttachId(), attachment.getBook(), attachment.getData(),
```

Comment:
Why return a copy of the attachment here?

Reply:
Consider using lombok to make a copy

Reply:
Marked with @Value